src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs

```diff
@@ -1201,7 +1201,7 @@ static internal void RootCallback(int id, int returnType, uint frameNumber, ref
 
             if (weakReferencesOfLottie.TryGetValue(id, out current))
             {
-                if (current.TryGetTarget(out currentView))
+                if (current.TryGetTarget(out currentView) && !currentView.Disposed && !currentView.IsDisposeQueued)
                 {
                     if (currentView != null && currentView.InternalSavedDynamicPropertyCallbacks != null &&
```

Comment:
요 바로 아랫줄에 `currentView != null` 체크가 있는데요, 이번에 추가된 조건과 상충해서요. 
```suggestion
                if (current.TryGetTarget(out currentView) && currentView != null && !currentView.Disposed && !currentView.IsDisposeQueued)
                {
                    if (currentView.InternalSavedDynamicPropertyCallbacks != null &&
```

Reply:
코멘트 감사드립니다.
제안해주신 방식으로 수정하였습니다.